hack/processing-plan.yaml

```diff
@@ -582,11 +582,19 @@ files:
     operations:
       - type: UPDATE
         key: .spec.template.spec.containers[0].image
-        value: '"{{ .Values.kueueViz.backend.image | default \"us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueueviz-backend:main\" }}"'
+        value: '"{{ .Values.kueueViz.backend.image.repository }}:{{ .Values.kueueViz.backend.image.tag | default .Chart.AppVersion }}"'
+        onFileCondition: '.metadata.name == "kueueviz-backend"'
+      - type: UPDATE
+        key: .spec.template.spec.containers[0].imagePullPolicy
```

Comment:
This looks suspicious, why do we update the imagePullPolicy both in processing-plan and hack/push-chart.sh?

Reply:
Not sure I got what you mean. In processing-plan.yaml, we're updating the template, and in push-chart.sh, we're updating the value.

Reply:
the aim of the task is to align how the image is handled for the manager and kueueviz, but for manager. For manager I don't see us updating imagePullPolicy in the processing-plan.yaml - only in the push-chart.sh